run-hb-pytest.sh

```diff
@@ -9,4 +9,4 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )"
 
-make -C $DIR/hammerblade/torch clean test_loader.cosim.log
+make -C $DIR/hammerblade/torch clean regression
```

Comment:
Could you add `make -C $DIR/hammerblade/torch clean clean` to the end? I think after running this script, the work directory is no longer clean due to untracked files under hammerblade/torch

Reply:
Sure, I always go inside `torch/` and run cosim followed by manual `make clean`. So, didn't notice this ;-).

Reply:
appreciate it!

Reply:
You can go ahead and merge once Travis finishes running ;)

Reply:
Yes, of course.